dowhy/causal_model.py

```diff
@@ -459,4 +460,55 @@ def summary(self, print_to_stdout=False):
             print(summary_text)
         return summary_text
 
+    def refute_graph(self, k= 1, method_name =None ):
```

Comment:
for full flexibility, let us allow user to specify which edges they want to check. As an optional parameter, you can ask user to provide a list of 3-tuples. e.g., ([['a'], ['b'], ['c'], ['a'], ['b'], ['d']]) where the task is to check independence of a and b conditional on c.